patientsearch/api.py

```diff
@@ -282,7 +281,7 @@ def update_resource_by_id(resource_type, resource_id):
     token = validate_auth()
 
     try:
-        resource = json.loads(request.headers.get("body", ""))
+        resource = request.get_json()
```

Comment:
@pbugni @achen2401
After clicking **View**, I'm getting a `400` on shared testing when PUT is called to search/create a patient.

I don't see any differences in the request from the frontend to backend, so I suspect this change is responsible, but I'm not sure why (`Content-type: application/json` is correctly included)

```
Bad Request

The browser (or proxy) sent a request that this server could not understand.
```

![image](https://user-images.githubusercontent.com/5333433/148848746-42cd569c-58db-40e3-a777-959aa28c48e2.png)

Reply:
This change is correct, we had old dysfunctional code there, from the initial implementation of this API as far as I can tell.

Should never be naming or requesting a **header** named **body** - the body of the request is correct, and where we should look for the attached JSON; just an implementation confusion.  The front end code has also been corrected to match (but isn't live where you saw this error on shared testing).

Reply:
Thanks for the explanation! looks good on shared testing now